FirebaseDatabase/Sources/Core/FRepo.m

```diff
@@ -534,26 +534,49 @@ - (void)getData:(FIRDatabaseQuery *)query
         return;
     }
     [self.persistenceManager setQueryActive:querySpec];
```

Comment:
I don't think we need this, since we already are setting a query as active when we create an event registration: 
https://github.com/firebase/firebase-ios-sdk/blob/eb30b6fdd809cbfe3a9eb7f3ccf1dd46326bef2d/FirebaseDatabase/Sources/Core/FSyncTree.m#L468